isis/src/qisis/objs/MosaicSceneWidget/GridGraphicsItem.cpp

```diff
@@ -193,7 +193,14 @@ namespace Isis {
 
               double x = 0;
               double y = 0;
-              bool valid = tproj->SetUniversalGround(lat.degrees(), lon.degrees());
+              
+              bool valid;
+              if (mappingGroup["LongitudeDirection"][0] == "PositiveWest") {
+                valid = tproj->SetGround(lat.degrees(), lon.positiveWest(Angle::Degrees));
```

Comment:
Changed `setUniversalGround` to `setGround`. `setUniversalGround` tries to naively account for a positive west angle if the projection object is set to `positiveWest`, although the grid code expects exclusively positive east angles. This basically keeps everything in positive east angles until the set ground call and lets the `Longitude` object handle angle conversion.

Reply:
Can you put a comment in the code that briefly says this?

Reply:
This is more of a comment on the previous state and how inexplicit it was than the current. I can put a comment on the swap mentioning the forced swap and maybe a minor comment of why we are explicitly converting but the latter seems like a low-value comment.

Reply:
I think it's valuable to document that the grid code expects exclusively positive east angles. Maybe this should just be an issue?

Reply:
I think it's worth making an issue as to why lons are incorrect when passed into the constructor on positive west so we can avoid the awkward min/max lon swap.  Preference for positive east lons seems to be common in ISIS?